components/kubearchive/base/kubearchive-config.yaml

```
@@ -6,3 +6,8 @@ metadata:
   namespace: product-kubearchive
 spec:
   resources:
+    - selector:
+        apiVersion: appstudio.redhat.com/v1alpha1 
+        kind: Snapshot
+      archiveOnDelete: true
```

The type is `string` because its a CEL expression, so it should be quoted.

```suggestion
      archiveOnDelete: 'true'
```

---

I can't resolve the comment, not sure if you can.